src/main/java/com/dre/brewery/utility/BoundingBox.java

```diff
@@ -20,26 +20,45 @@
 
 package com.dre.brewery.utility;
 
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
 import org.bukkit.Location;
 import org.bukkit.block.Block;
 
 import java.util.List;
 
+@Getter
+@Setter
 public class BoundingBox {
 
-    private int x1, y1, z1, x2, y2, z2;
+    @Data
+    @AllArgsConstructor
+    public static class BlockPos {
+        private int x, y, z;
+    }
```

Comment:
I prefer records, but I guess BreweryX uses lombok, so I don't know what's best there. Might be a good idea to make these values final though. I've also never liked the multiple declaration of variables in one line like that kind of statement, but who cares.